### PR TITLE
Make sure Gem works with Rails 5.

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,8 @@
 sudo: false
 cache: bundler
 rvm:
-  - 2.2
+  - 2.2.4
   - 2.3.0
-  - jruby
-  - rbx-2
```
Comment (tgxworld) [inline .travis.yml:7]: Couldn't install Rack 2.0.0 alpha. Remove them since Rails repo is not including them as well.

**Comment by tgxworld** [comment]
@rafaelfranca Please review :smile:  Thanks!

**Comment by tgxworld** [comment]
We'll need to bump the version and publish a new Gem for this. Is that ok?

**Comment by rafaelfranca** [comment]
Lets fix the other issue first.

**Pull request merged**